app/client/src/components/pages/State/components/Documents.js

```diff
@@ -179,7 +188,7 @@ function Documents({
   return (
     <Container>
       <h3>Documents Related to Integrated Report</h3>
-
+      <em>Select a document below to download a copy.</em>
```

Comment:
```suggestion
      <em>Select a document below to download a copy of the report.</em>
```

The ticket has slightly different text. Can you update this text?

Reply:
Updated, thanks!